src/components/apps/ids.js

```diff
@@ -8,6 +8,8 @@ export default {
     FAVORITES: "favorites",
     APP_COMMENTS: "appComments",
     CARD: "card",
+    CLOSE_BTN: "closeButton",
```

Comment:
GitHub Actions is displaying a `Duplicate key` warning below because `CLOSE_BTN` was already there.